hack/run-latency-testing.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
-GINKGO_SUITS=${GINKGO_SUITS:-"./test/e2e/performanceprofile/functests/0_config ./test/e2e/performanceprofile/functests/5_latency_testing"}
+GINKGO_SUITS=${GINKGO_SUITS:-"test/e2e/performanceprofile/functests-performance-profile-creator"}
```

Comment:
why did the path change here from ../5_latency_testing to ../functests-performance-profile-creator ?

Reply:
ouh copy paste  error :S